plugins/flytekit-modin/setup.py

```diff
@@ -0,0 +1,35 @@
+from setuptools import setup
+
+PLUGIN_NAME = "modin"
+
+microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
+
+plugin_requires = ["flytekit>=0.16.0b6,<1.0.0", "modin", "ray>=1.4.1"]
+
+__version__ = "0.0.1"

```

Comment:
This should be a template string actually - there's a CI job that will fill it in on release.  See [pandera](https://github.com/flyteorg/flytekit/blob/00d0c4bf00d63b9a67995e949e56b92d094015a0/plugins/flytekit-pandera/setup.py#L9) as an example.